bundled/mesquite-2.99/CMakeLists.txt

```diff
@@ -26,6 +27,14 @@ STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wformat")
 STRIP_FLAG(DEAL_II_CXX_FLAGS "-Woverloaded-virtual")
 STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wformat-invalid-specifier")
 
+STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wpointer-arith")
+STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wswitch -Wsynth")
+STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wwrite-strings")
+STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wno-deprecated-declarations")
+STRIP_FLAG(DEAL_II_CXX_FLAGS "-Qunused-arguments")
+STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wno-unsupported-friend")
+STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wno-parentheses")
+STRIP_FLAG(DEAL_II_CXX_FLAGS "-Wno-unused-local-typedefs")
```

Comment:
@tamiko I stripped out all of our additional build flags, but somehow they sometimes still find their way into the build command on some of the testers (and on my local machine too). What is the correct way to ensure that the bundled package is built without any warnings being emitted, or warnings being converted to build errors?